examples/simple_example_access_level_dry_run/variables.tf

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "parent_id" {
+  description = "The parent of this AccessPolicy in the Cloud Resource Hierarchy. As of now, only organization are accepted as parent."
+  type        = string
+}
+
+variable "policy_name" {
+  description = "The policy's name."
+  type        = string
+}
+
+variable "protected_project_ids" {
```

Comment:
Is there any need to collect IDs? They're not used in the example.

Also this is collecting a single project ID so should be named accordingly (or logic should be updated).

Reply:
I think this is copied verbatim from the other test (`simple_example_access_level`). I can fix it though...

Reply:
It'd be good to take them out from both if they're not used.

Reply:
They are used in setting up the access level protected resources. But I'll change the plural to singular on both.